lib/src/helpers.dart

```diff
@@ -180,3 +206,83 @@ num convertArea(num area,
 
   return (area / startFactor) * finalFactor;
 }
+
+
+/// Converts coordinates from one system to another.
+///
+/// Valid systems: [CoordinateSystem.wgs84], [CoordinateSystem.mercator]
+/// Returns: [List] of coordinates in the target system
+List<double> convertCoordinates(
```

Comment:
we have this special type `Position` that we use for coordinates (that implements a few extra utilities (like some maths) for coordinates. Can we use this instead of `List<double>`